.gitignore

```diff
@@ -29,7 +29,7 @@ build/Release
 # Dependency directories
 node_modules
 jspm_packages
-package-lock.json
+yarn.lock
```

Comment:
eh?

Reply:
The original package only had a `yarn.lock` (packages not installed by npm), but since all our frontend libraries tend to use `npm ci`, I switched this repo to not utilize anything to do with yarn anymore :)